priv/chttpd_handler.cfg

```diff
@@ -0,0 +1,47 @@
+%% Licensed under the Apache License,  Version 2.0 (the "License");  you may not
+%% use this file except in compliance with the License. You may obtain a copy of
+%% the License at
+%%
+%%                 http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless  required  by  applicable  law  or  agreed  to  in  writing,  software
+%% distributed  under the  License  is distributed on an "AS IS" BASIS,  WITHOUT
+%% WARRANTIES  OR  CONDITIONS  OF  ANY  KIND, either express or implied. See the
+%% License for the specific language governing permissions and limitations under
+%% the License.
+
+%% @doc link endpoints to callbacks - configs of multiple deps are combined.
+
+{docs, clauses, [
+    {"",                      {chttpd_misc, handle_welcome_req, 1}},
+    {"favicon.ico",           {chttpd_misc, handle_favicon_req, 1}},
+    {"_utils",                {chttpd_misc, handle_utils_dir_req, 1}},
+    {"_all_dbs",              {chttpd_misc, handle_all_dbs_req, 1}},
+    {"_active_tasks",         {chttpd_misc, handle_task_status_req, 1}},
+    {"_node",                 {chttpd_misc, handle_node_req, 1}},
+    {"_reload_query_servers", {chttpd_misc,
+        handle_reload_query_servers_req, 1}},
+    {"_replicate",            {chttpd_misc, handle_replicate_req, 1}},
+    {"_uuids",                {chttpd_misc, handle_uuids_req, 1}},
+    {"_session",              {chttpd_auth, handle_session_req, 1}},
+    {"_up",                   {chttpd_misc, handle_up_req, 1}},
+    {"_user",                 {chttpd_auth, handle_user_req, 1}},
+    {'_',                     {chttpd_db,   handle_request, 1}}
```

Comment:
Seems like "_oauth" accidentally missed, Can we also keep these lists sorted by resource name?